src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF1003.UrisMustBeValid_Invalid.sarif

```diff
@@ -1,38 +1,55 @@
 {
-  "$schema": "ht%tp://json.schemastore.org/sarif-2.0.0",
-  "version": "2.0.0",
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
   "runs": [
     {
       "tool": {
+        "driver": {
           "name": "CodeScanner",
           "downloadUri": "ht%tp://www.example.com/tools/codescanner/download.html",
-          "rules": [
-            {
-              "id": "TST0001",
-              "helpUri": "ht%tp://www.example.com/rules/tst0001.html"
-            }
-          ],
           "notifications": [
             {
               "id": "MSG0001",
-              "helpUri": "ht%tp://www.example.com/rules/msg0001.html"
+              "helpUri": "ht%25tp://www.example.com/rules/msg0001.html"
```

Comment:
>%25 [](start = 28, length = 3)

seems like a strange change to introduce, i am sure you had a reason to switch this to %25. just curious?

Reply:
Again, a result of `sarif transform`. I don't know whether it's `System.Uri` or Newtonsoft that does it, but during reserialization, characters that are invalid in a URI are percent-escaped. But that renders the URI valid, which is exactly what we don't want for this test.

Reverting. Good catch.

---
In reply to: [434245424](https://github.com/microsoft/sarif-sdk/pull/1898#discussion_r434245424) [](ancestors = 434245424)